docs/reference/deprecated_api_list.rst

```diff
@@ -6,88 +6,174 @@
 HIP deprecated runtime API functions
 **********************************************************************************************
 
-Several of our API functions have been flagged for deprecation. Using the following functions results in
-errors and unexpected results, so we encourage you to update your code accordingly.
+Several of our API functions have been flagged for deprecation. Using the
+following functions results in errors and unexpected results, so we encourage
+you to update your code accordingly.
 
-Context management
+Deprecated since ROCm 1.6.0
```

Comment:
I like the addition of the Deprecated release, but best to reverse sort this from most recent to oldest? Also, isn't there some rule about how long things can be deprecated? 

I also like the addition of the link to the feature documentation.

Reply:
Thanks Randy! I reversed the order or releases.

It is a good question about how long functions needs to be deprecated.